calcom/cal.com
# feat: add matomo analytics app

## What does this PR do?

This PR is to add Matomo analytics in the app store

Fixes #12136 

Loom Video: https://www.loom.com/share/24882809f4f241098a3cb84c688a2c2d?sid=ac2ba341-3011-4c33-9a79-a7e5bd96e826

## Requirement/Documentation



- If there is a requirement document, please, share it here.
- If there is ab UI/UX design document, please, share it here.

## Type of change




## How should this be tested?

* Add Matomo analytics application to an event type.
* Open up the event page and it should automatically track it.

## Mandatory Tasks
